### PR TITLE
Issue-3685: Variable based separator used for fullyQualifiedName instead of hardcoded . for Python and make the separator to :

### PR DESCRIPTION
**Pull request by harshach**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: ingestion/examples/sample_data/datasets/tables.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
     {
       "id": "3cda8ecb-f4c6-4ed4-8506-abe965b54b86",
       "name": "dim_address",
-      "fullyQualifiedName": "bigquery_gcp.shopify.dim_address",
```
Comment (Sachin-chaurasiya) [inline ingestion/examples/sample_data/datasets/tables.json:6]: @harshach, we have removed the `fullyQualifiedName`, is this change is intended?

**Review by Sachin-chaurasiya — COMMENTED** [review]
Minor comment.

**Review by ayush-shah — APPROVED** [review]

File: ingestion/examples/sample_data/datasets/tables.json (hunk 1 above)
Comment (ayush-shah) [reply ingestion/examples/sample_data/datasets/tables.json:6]: Hey Sachin - this shouldn't affect UI - Ingestion will create a FQN using FQN separator itself.

**Review by pmbrull — APPROVED** [review]

File: ingestion/examples/sample_data/datasets/tables.json (hunk 1 above)
Comment (harshach) [reply ingestion/examples/sample_data/datasets/tables.json:6]: this is not needed for ingestion to carry-forward. Backend will ignore any FQN being sent as part of createTableRequest and add its own

**Comment by pmbrull** [comment]
Checking python CI

**Comment by ayush-shah** [comment]
I am merging this PR
@kshinde2512 - can you create separate issue on to track and fix the selenium tests

**Pull request merged**